tutorials/lsrro.ipynb

```diff
@@ -0,0 +1,5428 @@
+       "      <td>7.846733</td>\n",
+       "      <td>51.132170</td>\n",
+       "      <td>46.008930</td>\n",
+       "      <td>8.032401</td>\n",
+       "      <td>51.765700</td>\n",
+       "      <td>46.627340</td>\n",
+       "      <td>20.0</td>\n",
+       "      <td>65.987580</td>\n",
+       "      <td>46.008930</td>\n",
+       "      <td>20.82099</td>\n",
+       "      <td>66.786150</td>\n",
+       "      <td>46.627340</td>\n",
+       "      <td>1.512</td>\n",
+       "      <td>10.27290</td>\n",
+       "      <td>12.94002</td>\n",
+       "      <td>0.126</td>\n",
+       "      <td>14.45138</td>\n",
+       "      <td>28.88252</td>\n",
+       "      <td>11.47307</td>\n",
+       "      <td>18.47401</td>\n",
+       "      <td>18.134540</td>\n",
+       "      <td>11.48456</td>\n",
+       "      <td>965.0740</td>\n",
+       "      <td>1738.4210</td>\n",
+       "      <td>1.341304</td>\n",
+       "      <td>1.878622</td>\n",
+       "      <td>1.880906</td>\n",
+       "      <td>213.9737</td>\n",
+       "      <td>157.52690</td>\n",
+       "      <td>150.15930</td>\n",
+       "      <td>195.84940</td>\n",
+       "      <td>74.19928</td>\n",
+       "      <td>78.99558</td>\n",
+       "      <td>0.219291</td>\n",
+       "      <td>0.162726</td>\n",
+       "      <td>0.161136</td>\n",
+       "      <td>0.202140</td>\n",
+       "      <td>0.079623</td>\n",
+       "      <td>0.087066</td>\n",
+       "      <td>15.553100</td>\n",
+       "      <td>8.949656e-08</td>\n",
+       "      <td>5.161416e-01</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>24.113060</td>\n",
+       "      <td>8.088381</td>\n",
+       "      <td>-4.870929</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>-5.531870</td>\n",
+       "      <td>10.682170</td>\n",
+       "      <td>24.113060</td>\n",
+       "      <td>3.072653</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>16</th>\n",
+       "      <td>130.0</td>\n",
+       "      <td>0.275</td>\n",
+       "      <td>8.174869</td>\n",
+       "      <td>2.248089</td>\n",
+       "      <td>34.638050</td>\n",
+       "      <td>9.525464</td>\n",
+       "      <td>3.0</td>\n",
+       "      <td>178.33070</td>\n",
+       "      <td>1000.0</td>\n",
+       "      <td>28401.84</td>\n",
+       "      <td>7810.506</td>\n",
+       "      <td>41.77572</td>\n",
+       "      <td>-7.484046</td>\n",
+       "      <td>34.291670</td>\n",
+       "      <td>17.91482</td>\n",
+       "      <td>28.62886</td>\n",
+       "      <td>99.23403</td>\n",
+       "      <td>692.40090</td>\n",
+       "      <td>3.937162</td>\n",
+       "      <td>4.237707</td>\n",
+       "      <td>0.905938</td>\n",
+       "      <td>0.559396</td>\n",
+       "      <td>0.081983</td>\n",
+       "      <td>0.421263</td>\n",
+       "      <td>1.968581</td>\n",
+       "      <td>2.424664</td>\n",
+       "      <td>0.631895</td>\n",
+       "      <td>1.181149</t
```

Comment:
Shall we remove all rows with NA values?

Reply:
I don't think we need to